Makefile

```diff
@@ -37,9 +37,6 @@ clean::
 
 # -------------------- BACKEND --------------------
 
-PART = xc7a35tcsg324-1
-#PART = xc7a50tfgg484-1
-
```

Comment:
I would rather keep default part in makefile. You can remove commented-out line and move default one to the top of the file and then change assign operator to `?=` like this: `PART ?= xc7a35tcsg324-1`
(reference https://www.gnu.org/software/make/manual/html_node/Setting.html)

Reply:
Okay! I put in a line for the default part as you suggested.